pkg/maps/lbmap/lbmap.go

```diff
@@ -79,6 +79,8 @@ type UpsertServiceParams struct {
 	CheckSourceRange          bool
 	UseMaglev                 bool
 	L7LBProxyPort             uint16 // Non-zero for L7 LB services
+	Name                      string // Name of the k8s Service
+	Namespace                 string // Namespace of the k8s Service
```

Comment:
`pkg/loadbalancer` already has `type ServiceID uint16`.

Reply:
Ended up moving `type Name struct` from `pkg/service` to `pkg/loadbalancer` as `type ServiceName struct` and refactoring the code to use it. Even though it currently is the k8s definition of a service name I think it makes sense to keep it separate since loadbalancing can be used outside k8s and we might want to tweak how we deal with names in the future.